tests/functional/adapter/seed_column_types/fixtures.py

```diff
@@ -111,4 +110,4 @@
                 amount: ["integer", "number"]
                 amount_usd: ["decimal", "number"]
                 created: ["timestamp", "string"]
-"""
+""".strip()
```

Comment:
Like my mom used to say when she was bundling me up for school, "Remember to strip your whitespace on seeds, dear"